doc/src/overrides.md

```diff
@@ -91,9 +91,14 @@ profile = "minimal"
 The `[toolchain]` section is mandatory, and at least one property must be
 specified. `channel` and `path` are mutually exclusive.
 
-For backwards compatibility, `rust-toolchain` files also support a legacy
+For backwards compatibility, the `rust-toolchain` file also supports a legacy
```

Comment:
this change isn't correct as I understand english grammar. `files` is correct because we are not talking about a single instance but about the class, like `people`. `people support a government` would be an similar construction.